src/convertToObject.js

```diff
@@ -6,7 +6,15 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const objStyles = {};
+
+  sourceString.split(';').map((style) => {
```

Comment:
The code is splitting the string by semicolons, which is correct, but it is not handling empty or invalid declarations. You should filter out empty strings after splitting to avoid creating empty or undefined keys in the object.